src/components/Modal/index.js

```diff
@@ -44,13 +44,12 @@ export default class Modal extends Component {
             this.contentElement = this.contentRef.current;
             CounterManager.increment();
             disableBodyScroll(this.contentRef.current);
-            this.modalTriggerElement = document.activeElement;
             this.modalRef.current.focus();
         }
     }
 
     componentDidUpdate(prevProps) {
-        const { isOpen, onOpened } = this.props;
+        const { isOpen, onOpened, modalTriggerElement } = this.props;
```

Comment:
we should avoid adding props since this change the component API and this needs to be discussed carefully, we should try to find an internal solution about how to find the active element

Reply:
Ok, I will look for another soluton them, thank you